scst-store/create-service-account.hbs.md

```diff
@@ -84,7 +75,7 @@ As a part of the Store installation, the `metadata-store-read-only` cluster role
     EOF
     ```
 
-    >**Note:** For Kubernetes v1.24 and later, services account secrets are no longer automatically created, the service account secret must be manually created.
+  >**Note:** For Kubernetes v1.24 and later, services account secrets are no longer automatically created, the service account secret must be manually created.
 
 If you do not want to bind to a cluster role, create a read-only role in the `metadata-store` namespace with a service account. The following example command creates a service account named `metadata-store-read-client`, depending on the Kubernetes version:
```

Comment:
Reading this whole file, it feels like there are a lot of code blocks. Do you think it would help to break it up with a subheader for this section (like `With Custom Cluster Role Binding`)?